Automattic/sensei
# Menu should not collapse when navigated to Messages.

Fixes https://github.com/Automattic/sensei/issues/1638.

Test case:
1. Navigate to Sensei in the Admin panel
2. Navigate to Messages
Expected: The parent menu (Sensei) should not collapse and Messages should be highlighted <img width="572" alt="screen shot 2016-11-12 at 17 15 00" src="https://cloud.githubusercontent.com/assets/1620929/20239205/8f630076-a8fb-11e6-84d4-2af4cf5385f4.png">
Actual: The parent menu collapses <img width="573" alt="screen shot 2016-11-12 at 17 14 51" src="https://cloud.githubusercontent.com/assets/1620929/20239204/8f5a7a0a-a8fb-11e6-9f1a-96454e1c61c8.png">

Repeat test case for all root menus (Sensei, Courses, Lessons, Questions) and their corresponding items.
